### ISSUE TITLE
`ctrl+?` to see keyboard shortcuts

### ISSUE THREAD

**Issue by ShadowJonathan** [ajbura/cinny#219]
This is almost standard across all messengers (or at least on discord, slack, and element) to see all normal keyboard shortcuts on the messenger itself with.

**Comment by vwkd** [comment]
Maybe worth considering if this is necessary at all if there were only a single keyboard shortcut for everything. How could that possibly be? A Command Palette… #129

**Comment by BachoSeven** [comment]
@vwkd I still think that the keyboard shortcuts such as the upwards arrow proposed in https://github.com/ajbura/cinny/issues/97 should _not_ be placed behind another "single keyboard shortcut" as they are pretty essential and should be snappy, so that it would be good to document them through `Ctrl+?` 

Edit: OP probably wasn't talking about the command-line commands that you propose to unify under a command palette, I think, but rather the keyboard shortcuts like `Tab` for navigating through Emoji suggestions, the aforementioned Upwards arrow etc.

**Comment by kfiven** [comment]
Also https://github.com/cinnyapp/cinny/issues/1189#issuecomment-1577510978

As far as I know there is no shortcut for the emoji panel either. We should add the sticker and emoji shortcutst together. ctrl + s and ctrl + e respectively
